### PR TITLE
🌐 Add Portuguese translation for `docs/pt/docs/tutorial/dependencies/global-dependencies.md`

### PR DESCRIPTION
**Pull request by Joao-Pedro-P-Holanda**

### CODE REVIEW

**Comment by tiangolo** [comment]
📝 Docs preview for commit c9bb13dc6fc6f4ba37598aad628836ae50fe74e2 at: https://60e4b409.fastapitiangolo.pages.dev

File: docs/pt/docs/tutorial/dependencies/global-dependencies.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# Dependências Globais
+
+para alguns tipos de aplicação específicos você pode querer adicionar dependências para toda a aplicação.
```
Comment (ceb10n) [inline docs/pt/docs/tutorial/dependencies/global-dependencies.md:3]: the capital letter is missing at the beginning

**Review by ceb10n — CHANGES_REQUESTED** [review]
Great translation! 👏

Just the first letter from the first sentence that is missing the capital letter

File: docs/pt/docs/tutorial/dependencies/global-dependencies.md (hunk 1 above)
Comment (Joao-Pedro-P-Holanda) [reply docs/pt/docs/tutorial/dependencies/global-dependencies.md]: Thanks for noticing :)

**Comment by tiangolo** [comment]
📝 Docs preview for commit 187bd7a3e63aee7036cfcbbdeb7cc6868c551837 at: https://24ec6553.fastapitiangolo.pages.dev

**Review by ceb10n — APPROVED** [review]

**Review by lucasbalieiro — APPROVED** [review]
Nice! Seems fine to me.

**Comment by alejsdev** [comment]
Cool :sunglasses: 
Thanks for the help! @Joao-Pedro-P-Holanda @ceb10n @lucasbalieiro :v:

**Pull request merged**